sanic/cookies.py

```diff
@@ -109,14 +203,17 @@ def __setitem__(self, key, value):
         if key not in self._keys:
             raise KeyError("Unknown cookie property")
         if value is not False:
```

Comment:
This might be a good place to handle `None` values, meaning to `del` the parameter. Currently it converts None to string "None" for most parameters.

Reply:
I am doing this a slightly different way in next commit push.